src/mds/Migrator.h

```diff
@@ -89,7 +89,6 @@ class Migrator {
     set<mds_rank_t> warning_ack_waiting;
     set<mds_rank_t> notify_ack_waiting;
     map<inodeno_t,map<client_t,Capability::Import> > peer_imported;
-    list<MDSInternalContextBase*> waiting_for_finish;
```

Comment:
"mds: remove useless code in Migrator"
With commits like that it's best to say exactly why it's useless. In this case, I gather nobody ever put anything on waiting_for_finish or called add_export_finish_waiter(), but the commit should say so.